luigi/server.py

```diff
@@ -73,6 +92,62 @@ def get_template_path(self):
         return pkg_resources.resource_filename(__name__, 'templates')
 
 
+class AllRunHandler(BaseTaskHistoryHandler):
+
+    def get(self):
+        # get task history db in the same path
+        engine = create_engine('sqlite:///.task_history.db', echo=True)
+        conn_sqlite = engine.connect()
+        query = "select id, name from tasks order by id desc"
+        task_list = conn_sqlite.execute(query)
+        tasknames = []
+        for task in task_list:
+            tasknames.append(task['name'])
+        self.render("menu.html", tasknames=tasknames)
+
+
+class SelectedRunHandler(BaseTaskHistoryHandler):
+
+    def get(self, name):
+        tasks = {}
+        chartResults = {}
+        tableResults = []
+        # get task history db in the same path
+        engine = create_engine('sqlite:///.task_history.db', echo=True)
+        conn_sqlite = engine.connect()
+        task_query = "select id, name from tasks"
+        history_query = "select id, task_id, event_name, ts from task_events"
+        task_list = conn_sqlite.execute(task_query)
+        task_history = conn_sqlite.execute(history_query)
+        for task in task_list:
+            task_seq = task['id']
+            task_name = task['name']
+            tasks[task_seq] = str(task_name)
+        for task in task_history:
```

Comment:
Oh, this was because I thought it's useful to see the name of the task in addition to its task id. (e.g. The task named "Loader" can have event named "Done", "Failed", and "Running.")